src/utils/windows.js

```diff
@@ -58,7 +58,7 @@ export const createAttributes = attrs => ({
   visibility: 'global',
   shadowDOM: false,
   clamp: true,
-  droppable: true,
+  droppable: {},
```

Comment:
There's no need to refactor the default value here since you're introducing a new type.

Reply:
Sorry, I didn't notice this one in my previous review.

Reply:
✋ 
74c82667bb1f0d5d4994212b1fe827bff40d82ca